src/lsm/manifest.zig

```diff
@@ -342,39 +346,37 @@ pub fn ManifestType(comptime Table: type, comptime Storage: type) type {
             assert(level < config.lsm_levels);
             assert(compare_keys(key_min, key_max) != .gt);
 
+            const direction = .ascending;
```

Comment:
Doesn't this need to be `descending` to prevent the iterator getting out of sync when tables are removed?

Reply:
Yea, forgot about that discussion. Although I believe that the tables recorded into the buffer will need to be reversed before calling `remove_tables()` as that expects them to be ascending

Reply:
Good point! Yes, it does need to be reversed.